cli/spinner.ts

```diff
@@ -19,10 +19,11 @@ const DEFAULT_SPINNER = ["⠋", "⠙", "⠹", "⠸", "⠼", "⠴", "⠦", "⠧",
 export type Ansi = string & {};
 
 /**
- * **UNSTABLE**: New API, yet to be vetted.
- *
  * Color options for {@linkcode SpinnerOptions.color}.
  *
+ * > [!WARNING]
+ * > **UNSTABLE**: New API, yet to be vetted.
```

Comment:
Note: this doesn't look the best when hovered-on in VSCode, but I think that's fixable, and a nit, compared to how prominent these notices are [in JSR](https://jsr.io/@std/net@1.0.0-rc.1/doc/~/getNetworkAddress).

![image](https://github.com/user-attachments/assets/90c16209-f831-46f2-9cb7-d4e517eb6ca8)